tests/utils/PolygonUtilsTest.cpp

```diff
@@ -445,7 +462,9 @@ TEST_F(PolygonUtilsTest, RelativeHammingQuarterOverlap)
  */
 TEST_F(PolygonUtilsTest, RelativeHammingLineSquare)
 {
-    ASSERT_EQ(PolygonUtils::relativeHammingDistance(test_squares, test_line), 1.0) << "The difference between the polygons is 100% because the area of the difference encompasses the area of the one polygon that has area.";
+    ASSERT_EQ(PolygonUtils::relativeHammingDistance(test_squares, test_line), 1.0)
+      << "The difference between the polygons is 100% because the area of the difference encompasses the area of the one polygon that has "
```

Comment:
Indent with 4 spaces.

Why is "area" on a separate line? Seems to be harder to read.